src/dotnet/commands/dotnet-cache/LocalizableStrings.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace Microsoft.DotNet.Tools.Cache
+{
+    internal class LocalizableStrings
+    {
+        public const string AppFullName = ".NET Cache";
+
+        public const string AppDescription = "Caches Assemblies specified for the .NET Platform, by default these will be optimized for the target runtime and framework.";
```

Comment:
lowercase assemblies. end sentence after platform. missing comma after By default:
Caches assemblies specified for the .NET Platform. By default, ...

Also not sure if specified is in the right place here. Are you saying that the assemblies are specified for the .NET Platform or are you saying that the assemblies were specified? If the latter, it should be:

"Caches the specified assemblies ..."